### PR TITLE
LWDEV-10361 Implement cash in/ out tests

### PR DESCRIPTION
**Pull request by olpapchenko**

### CODE REVIEW

File: src/main/kotlin/com/lykke/me/test/client/socket/MeClientFactory.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import java.lang.IllegalArgumentException
+
+@Component
+class MeClientFactory {
+
+    @Autowired
+    private lateinit var meClient: MeClient
+
+    @Autowired
+    private lateinit var meSocketProtoBlockingClient: MeBlockingClient
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/me/test/client/socket/MeClientFactory.kt]: it'is better to rename this field to meBlockingClient

File: src/main/kotlin/com/lykke/me/test/client/tests/CashInOutTest.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import org.springframework.beans.factory.annotation.Autowired
+import java.math.BigDecimal
+
+@MeTest
+class CashInOutTest {
+    private companion object {
+        val TEST_RUN_COUNT = 10_000
+    }
+
+    private lateinit var meClient: MeClient
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/me/test/client/tests/CashInOutTest.kt]: must be `Autowired` ?

File: src/main/kotlin/com/lykke/me/test/client/socket/MeClientFactory.kt (hunk 1 above)
Comment (olpapchenko) [reply src/main/kotlin/com/lykke/me/test/client/socket/MeClientFactory.kt]: fixed

File: src/main/kotlin/com/lykke/me/test/client/tests/CashInOutTest.kt (hunk 2 above)
Comment (olpapchenko) [reply src/main/kotlin/com/lykke/me/test/client/tests/CashInOutTest.kt]: this field is wired by test runner - depending in the parameter value of input request - `messageRatePolicy`

**Pull request merged**